client/src/app/config/app.config.ts

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
it's great that you decided to create a config but angular provides a solution for that
pls read https://angular.io/guide/build
Configuring application environments

Reply:
I create environments folder in src folder, I understand the environment.ts file will be replaced by environments.prod.ts file in the building